src/options/smt_options.toml

```diff
@@ -200,6 +200,25 @@ header = "options/smt_options.h"
   type       = "bool"
   help       = "turn on unsat core generation"
 
+[[option]]
+  name       = "unsatCoresMode"
+  smt_name   = "unsat-cores-mode"
+  category   = "regular"
+  long       = "unsat-cores-mode=MODE"
+  type       = "UnsatCoresMode"
+  default    = "SAT_PROOF"
```

Comment:
Default should be ASSUMPTIONS eventually right?

Reply:
Definitely. When it's merged to master this should be updated.